src/components/Generator/GenerateImage.js

```diff
@@ -25,47 +30,48 @@ const GenerateImage = ({ onSubmit }) => {
       setTimeout(() => {
         setLoading(false);
         setImagesAreShown(true);
-      }, 5000);
+      }, 2000);
     }
 
-    const data = {
-      valueText: inputTextValue,
-    };
-
     onSubmit(data);
   };
 
+  const getType = (e) => {
+    setData({ valueText: inputTextValue, typeValue: e.target.innerHTML });
+  };
```

Comment:
Will try to add this info when the form is submitted so that we can get the data if the user first changes the type and then inputs the text